### ISSUE TITLE
Rive render opensource

### ISSUE THREAD

**Issue by PluckyDuck99** [kibble-cabal/godot-rive#15]
Now that Rive Render is open-source, do you have plans to integrate with it? I would really like to work on a project using Rive and Godot. Currently, they don't have official support, but your library does a great job. It would be great if you could port it to their Render, which reportedly has excellent performance according to them. [Rive Renderer](https://github.com/rive-app/rive-renderer)

**Comment by audse** [comment]
@PluckyDuck99 Yes, there is definitely plans in the works to integrate/replace Skia with Rive Renderer. No timeline yet, but hopefully within the next few weeks.

**Comment by PluckyDuck99** [comment]
They are recruiting developers to improve runtimes in some engines, Unity, Unreal, among others. Could you offer to help/work them in development? It would be amazing now to be able to use vectors in games with good performance thanks to the renderer. From what I've read, they also intend to delve into Godot officially.

**Comment by TranquilMarmot** [comment]
😅 I beg them for official Godot support every time I see them mention Unity or Unreal

**Comment by PluckyDuck99** [comment]
@audse Any news on it? I recently saw a discussion that Godot doesn't have everything exposed yet for rive integration :(
